ci/cron/src/Main.hs

```diff
@@ -261,3 +258,4 @@ main = do
             putStrLn "Updating versions.json & hidden.json"
             update_s3 temp_dir gh_versions
         reset_cloudfront
+        Exit.exitSuccess
```

Comment:
I’d prefer not to add this. It isn’t necessary now and if you shuffle things you might accidentally move this outside of `main` where it can be quite problematic or at least confusing. In other words, don’t make `main` more special than it is by relying on the fact that calling `exitSuccess` at the end doesn’t break things.

Reply:
Would you also remove it from line 246/249 then? I don't particularly care about the explicit exit, but I do think it's best if both branches are symmetric in that regard.

Reply:
Good point, didn’t see that it was there already. Yes, best to remove it there as well.

Reply:
Done.